lib/protocol/bep_extensions.go

```diff
@@ -266,24 +266,37 @@ func BlocksEqual(a, b []BlockInfo) bool {
 }
 
 func (f *FileInfo) SetMustRescan(by ShortID) {
-	f.LocalFlags = FlagLocalMustRescan
-	f.ModifiedBy = by
-	f.Blocks = nil
-	f.Sequence = 0
+	f.setLocalFlags(by, FlagLocalMustRescan)
 }
 
 func (f *FileInfo) SetIgnored(by ShortID) {
```

Comment:
`ShortID` is capitalized in all of these, old sin, but shouldn't be...

Reply:
Why? It's used abundantly outside of lib/protocol.

Reply:
braino